jni/tests/Foreign/JNISpec.hs

```diff
@@ -67,3 +69,12 @@ spec = do
           jniMethodToJavaSignature "(SF)V" `shouldBe` Right (["short", "float"], "void")
           jniMethodToJavaSignature "(C[Ljava/lang/String;D)Z"
             `shouldBe` Right (["char", "java.lang.String[]", "double"], "boolean")
+
+      describe "showException" $
+        it "correctly displays exceptions" $ do
+          kinteger <- findClass (referenceTypeName (sing :: Sing ('Class "java.lang.Integer")))
+          let sig = methodSignature [] (sing :: Sing 'Void)
+          result <- try $ Internal.getMethodID kinteger (fromChars "toString") sig
```

Comment:
Looks like a case for [shouldThrow](https://www.stackage.org/haddock/lts-16.22/hspec-expectations-0.8.2/Test-Hspec-Expectations.html#v:shouldThrow).

Reply:
Nah, disregard this comment.